src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativePaymentPaidPersonalisation.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.legalrepresentative;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition.*;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.HearingCentre;
+import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
+import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.CustomerServicesProvider;
+
+@Service
+public class LegalRepresentativePaymentPaidPersonalisation implements EmailNotificationPersonalisation {
+
+    private final String legalRepresentativePaymentPaidBeforeListingTemplateId;
+    private final String legalRepresentativePaymentPaidAfterListingTemplateId;
```

Comment:
it is still open question for Business, do we want to suspend further appeal processing or not. It might be refactoring needed once decision is made

Reply:
Yes, makes sense to suspend in which case "after listing" template is not required. May need to refactor once confirmation received.